### DISCUSSION TITLE
Ability to reorder tasks

### DISCUSSION THREAD

**Discussion by ryanaltvater** [rampatra/todobar-community#8] [Ideas]
Would be nice to have the ability to manually reorder the list of tasks, via drag and drop. That way you can rearrange according to grouping, relation, priority, etc.

**Comment by brandon-threinen** [comment]
This would be incredibly helpful. It's far too clunky to click into each task and adjust with the arrows considering you need to reselect the task if you make a single adjustment with the arrows. Would love to recommend this for my team but this is currently the limiting factor.
